ices/66805.sh

```diff
@@ -0,0 +1,137 @@
+#!/bin/bash
+
+tmp="$(mktemp -d)"
```

Comment:
glacier runs in a temp directory and handles cleanup, so mktemps can be removed

Reply:
This is a holdover from old code (fixed/59191.sh). Thanks for telling me!